packages/react/__tests__/get-label-from-stack-trace.js

```diff
@@ -0,0 +1,457 @@
+    at App.js:5
+    at Array.map (<anonymous>)
+    at MyComponent$9 (App.js:5)
+    at renderWithHooks (react-dom.development.js:14803)
+    at mountIndeterminateComponent (react-dom.development.js:17482)
+    at beginWork (react-dom.development.js:18596)
+    at beginWork$1 (react-dom.development.js:23179)`
+
+    expect(getLabelFromStackTrace(stackTrace)).toBe(expectedLabel)
+  })
+
+  test('Firefox', () => {
+    const stackTrace = `createEmotionProps@http://localhost:3000/static/js/main.chunk.js:844:40
+jsxDEV@http://localhost:3000/static/js/main.chunk.js:1126:247
+MyComponent$9/<.children<@http://localhost:3000/static/js/main.chunk.js:2274:106
+MyComponent$9@http://localhost:3000/static/js/main.chunk.js:2274:19
+renderWithHooks@http://localhost:3000/static/js/vendors~main.chunk.js:16616:31
+mountIndeterminateComponent@http://localhost:3000/static/js/vendors~main.chunk.js:19228:17
+beginWork@http://localhost:3000/static/js/vendors~main.chunk.js:20306:20
+beginWork$1@http://localhost:3000/static/js/vendors~main.chunk.js:24848:18`
+
+    expect(getLabelFromStackTrace(stackTrace)).toBe(expectedLabel)
+  })
+
+  test('Safari', () => {
+    const stackTrace = `createEmotionProps@http://localhost:3000/static/js/main.chunk.js:844:49
+jsxDEV@http://localhost:3000/static/js/main.chunk.js:1126:247
+map@[native code]
+MyComponent$9@http://localhost:3000/static/js/main.chunk.js:2274:22
+renderWithHooks@http://localhost:3000/static/js/vendors~main.chunk.js:16616:31
+mountIndeterminateComponent@http://localhost:3000/static/js/vendors~main.chunk.js:19228:32
+beginWork$1@http://localhost:3000/static/js/vendors~main.chunk.js:24848:27`
+
+    expect(getLabelFromStackTrace(stackTrace)).toBe(expectedLabel)
+  })
+
+  test('SSR', () => {
+    const stackTrace = `Error
+    at Object.createEmotionProps (webpack-internal:///../../packages/react/dist/emotion-element-7a9c77b4.cjs.dev.js:175:40)
+    at jsxDEV (webpack-internal:///../../packages/react/jsx-dev-runtime/dist/emotion-react-jsx-dev-runtime.cjs.dev.js:22:75)
+    at eval (webpack-internal:///./pages/index.js:78:99)
+    at Array.map (<anonymous>)
+    at MyComponent$9 (webpack-internal:///./pages/index.js:78:19)
+    at processChild (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3043:14)
+    at resolve (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:2960:5)
+    at ReactDOMServerRenderer.render (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3435:22)
+    at ReactDOMServerRenderer.read (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3373:29)
+    at Object.renderToString (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3988:27)`
+
+    expect(getLabelFromStackTrace(stackTrace)).toBe(expectedLabel)
+  })
+})
+
+/**
+ * E.g.
+ *
+ * ```
+ * const test = {
```

Comment:
it would be great to have such an example before each describe block - some of them are missing it and it makes it slightly harder to analyze.